### DISCUSSION TITLE
Stuck on Loading

### DISCUSSION THREAD

**Discussion by ShuggMcGlummfer** [NexovaDev/UpdateHub#205] [Help Thread] [answered]
Maybe I'm being too hasty, but after installing UpdateHub and running the exe file, it has stuck on Dashboard/Software health Loading... for 1hr. Is this normal behaviour? Going to Apps leaves it stuck on Checking for Updates...

Eventually got this error:-
Unhandled Error
TypeError: Invalid badgeNumber specified.
Expected: number
Got: object
    at Badge.update (C:/Users/Shugg McGlummfer/AppData/Local/Programs/update-hub/resources/app.asar/node_modules/electron-taskbar-badge/lib/cjs/index.js:66:10)
    at SystemPreferences.<anonymous> (C:/Users/Shugg McGlummfer/AppData/Local/Programs/update-hub/resources/app.asar/node_modules/electron-taskbar-badge/lib/cjs/index.js:50:9)
    at SystemPreferences.emit (node:events:513:28)

Thanks.

**Comment by onyx-nxt** [comment]


**Reply by sk00t3r802** [reply]
In version 2.2, or a later version?

**Comment by ShuggMcGlummfer** [comment]
I am now running 2.2 after downloading the exe file on the 2.2 release page, but it just looks the same to me, getting stuck on Loading. The About button is inactive, so I can't check the version. The frame does not have any border so there is no maximise/minimise/exit icons.

    On Wednesday, 13 September 2023 at 14:15:29 BST, sk00t3r802 ***@***.***> wrote:  
 
 


In version 2.2, or a later version?

**Reply by onyx-nxt** [reply]
can you send us your log file so we can debug the issue better?

**Reply by sk00t3r802** [reply]
Where can Ii find the log file? I am not seeing one in the application's directory (C:\Users\...\AppData\Local\Programs\update-hub).

**Reply by onyx-nxt** [reply]
it's located at `AppData/Roaming/update-hub` and it's called `app.log`

**Reply by sk00t3r802** [reply]
I don't have an app.log file in that directory. I have several log files under 'Session Storage' but those are the only .log files in the AppData\Roaming\update-hub directory. 
![image](https://github.com/KK-Designs/UpdateHub/assets/144707318/32328f25-c11b-4c21-87f8-f6dad6af1daa)

**Reply by onyx-nxt** [reply]
I think what's happening is that some error is happening that is preventing the app from loading completely, including the logger. Open the inspecting tool in the app by pressing <kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>*</kbd> and send a screenshot of the "console" tab.

**Comment by ShuggMcGlummfer** [comment]
Sorry CTRL + SHIFT + * does nothing.
Jim

    On Sunday, 17 September 2023 at 04:05:53 BST, KK-Designs ***@***.***> wrote:  
 
 


I think what's happening is that some error is happening that is preventing the app from loading completely, including the logger. Open the inspecting tool in the app by pressing CTRL + SHIFT + * and send a screenshot of the "console" tab.

**Reply by sk00t3r802** [reply]
I'm not able to get that to work either. I uninstalled, made sure the directories were removed, and reinstalled and still had the same behavior.

**Reply by onyx-nxt** [reply]
do you have WinGet installed?

**Reply by sk00t3r802** [reply, marked as answer]
I was positive I had WinGet, but when I typed winget into a command prompt, it gave me the "'winget' is not recognized as an internal or external command, operable program or batch file." error. After installing WinGet, UpdateHub works fine.

**Comment by ShuggMcGlummfer** [comment]
If I type winget into a Command Line I get the attached reply. Is this what you'd expect?
J Darroch

    On Friday, 6 October 2023 at 14:42:06 BST, KK-Designs ***@***.***> wrote:  
 
 


do you have WinGet installed?

**Reply by onyx-nxt** [reply]
I do not see the attachment, can you send it here?

**Comment by ShuggMcGlummfer** [comment]
It's a png.

-- 

/J //Darroch/
/Sent from my mobile/
----------------------------------------

10 Oct 2023 23:10:41 Nexova ***@***.***>:

**Reply by onyx-nxt** [reply]
I do not see that image, can you try uploading it again?

**Comment by ShuggMcGlummfer** [comment]
Attached, this time as jpg.
Jim

    On Wednesday, 11 October 2023 at 15:14:37 BST, Nexova ***@***.***> wrote:  
 
 


I do not see that image, can you try uploading it again?

**Reply by onyx-nxt** [reply]
I am still not able to see it, that's weird. Can you try just copy-pasting the command output here?

**Comment by ShuggMcGlummfer** [comment]
I'm away for a week so will get back to you in a bit.

-- 

/J //Darroch/
/Sent from my mobile/
----------------------------------------

12 Oct 2023 05:18:34 Nexova ***@***.***>:

**Comment by RebusCom** [comment]
I installed v2.3.4 on Win10 22H2 and have the same problem, stuck on loading or searching for updates. Computer is rebooted.  Winget is installed.  It's not a firewall issue.

**Reply by RebusCom** [reply]
Should a new thread be started?
